packages/babel-helper-module-transforms/src/rewrite-this.js

```diff
@@ -17,6 +17,11 @@ const rewriteThisVisitor = {
   ClassProperty(path) {
     path.skip();
```

Comment:
We have a method to skip everything but the computed key somewhere (it's used by `environmentVisitor`). We can re-use it here.

Reply:
Okay, so we can use that method inside the ClassProperty to find if the `ClassMethod` is computed or not ?

Reply:
We should do it both for class methods and for class properties

Reply:
Also, note that `ClassMethods` are already visited by the `Function` visitor: we need to handle them there.

Reply:
Ok. Thanks for this info. Adding to this, do we have to requeue the key somewhere (it was mentioned in the issue text)?

Reply:
Not skipping it should be enough. Currently skip+requeue is buggy.

Reply:
Ok, will send a patch commit asap. Exams starting tomorrow 😅

Reply:
Good luck!

Reply:
Thanks and same to you too 👍